addons/account/migrations/13.0.1.1/pre-migration.py

```diff
@@ -309,6 +309,7 @@ def migrate(env, version):
             WHERE key = 'sale.use_sale_note'"""
         )
     openupgrade.rename_models(cr, _model_renames)
+    openupgrade.remove_tables_fks(env.cr, ["account_invoice_payment_rel"])
```

Comment:
Shouldn't we should do this also with `account.invoice` and `account.invoice.line`?

Reply:
It's already done in base.

Reply:
:thinking: It will be probably be good to remove them in all three tables. Just a question, might be faster if FK exists for `account.invoice` and `acocunt.invoice.line`?

Reply:
Curious that it's done in base instead of here. No fk will be used until the load of this module. @etobella It's faster without FKs as no checks are performed.

Reply:
It's done in base because 2 reasons:
- historically, we have removed the FK for all obsolete tables in base, just to have them together in a place, to easily check them
- when the migration of base was done, we put all this account related tables already, we know we needed to delete their FKs. Also, it includes others like account_invoice_tax, vouchers and many2manys.

Reply:
It was done historically in base because previous your work on the migration logs, the only place obsolete models were logged was on base, but now obsolete models are also logged on each module. We can let it this version on base, but not this one here and the rest on base. Homogeneity.